.golangci.yml

```diff
@@ -102,6 +103,61 @@ linters:
           msg: Add a wrapper to nlwrap.Handle for EINTR handling and update the list in .golangci.yml.
       analyze-types: true
 
+    gocritic:
+      disabled-checks:
+        - appendAssign
+        - appendCombine
+        - assignOp
+        - badCall
```

Comment:
Probably easy fix; I _suspect_ the code tries to piggy-back on `filepath.Join` to get `filepath.Clean`, so probably can be swapped for that;

```
image/fs.go:147:13: badCall: suspicious Join on 1 argument (gocritic)
	baseDir := filepath.Join(s.metadataDir(dgst))
	           ^
```